### PR TITLE
feat(alert): add a tooltip to show the CCN title

### LINKED ISSUE #303: Alertes - ajouter en title le nom cour de la cc sur le titre
Dans le titre des alerts, il pourrait etre utile d'avoir le nom court de convention collective (ccn) en tooltip (dans un title)

### PR DESCRIPTION
**Pull request by m-maillot**
Closes #303

CleanShot 2021-02-26 at 09 43 20@2x

### CODE REVIEW

File: targets/frontend/src/pages/alerts/[[...params]].js [hunk 1]
```diff
@@ -93,13 +93,28 @@ export function AlertPage() {
...
-        alert.created_at
-      ).toLocaleDateString()} (${alert.ref})`;
+      return (
+        <>
+          IDCC {alert.info.num}
+          <Text
+            title={alert.info.title}
+            sx={{
+              fontVariantPosition: "super",
+              marginLeft: 4,
```
Comment (m-maillot) [inline targets/frontend/src/pages/alerts/[[...params]].js:103]: I'm not sure about this one. Theme-UI seems to use the value from the theme but I didn't saw margin values.

**Comment by m-maillot** [comment]
I can also show the `ccn` title on the entire line. I did that but I found more useful the icon ⓘ to draw the attention of the user.

File: targets/frontend/src/pages/alerts/[[...params]].js (hunk 1 above)
Comment (lionelB) [reply targets/frontend/src/pages/alerts/[[...params]].js:103]: Why not use an Icon here (with a aria-label) we use react-icons on the project

Comment (lionelB) [reply targets/frontend/src/pages/alerts/[[...params]].js:103]: also margin are expressed in small / medium / large

File: targets/frontend/src/pages/alerts/[[...params]].js [hunk 2]
```diff
@@ -93,13 +93,28 @@ export function AlertPage() {
...
-      return `IDCC ${alert.info.num} - ${new Date(
-        alert.created_at
-      ).toLocaleDateString()} (${alert.ref})`;
+      return (
+        <>
+          IDCC {alert.info.num}
+          <Text
+            title={alert.info.title}
+            sx={{
+              fontVariantPosition: "super",
```
Comment (UnbearableBear) [inline targets/frontend/src/pages/alerts/[[...params]].js:102]: Attention, le support de cette prop CSS est limitée à firefox. Tu peux utiliser vertical-align ou les balises html sup / sub pour plus de support

Comment (m-maillot) [reply targets/frontend/src/pages/alerts/[[...params]].js:102]: En effet, merci ! J'ai modifié par vertical align :)

File: targets/frontend/src/pages/alerts/[[...params]].js (hunk 1 above)
Comment (m-maillot) [reply targets/frontend/src/pages/alerts/[[...params]].js:103]: I've changed to react-icons:
CleanShot 2021-02-26 at 14 09 39@2x

**Comment by m-maillot** [comment]
@lionelB see with Florent, we keep the tooltip. The CC name is too long to be shown directly.

**Review by lionelB — APPROVED** [review]
Super! attention au rebase (ou merge) avant de terminer

**Pull request merged**